src/App/App.js

```diff
@@ -14,10 +14,36 @@ class App extends Component {
         console.log(this.state.movies)
     }
 
+    componentDidMount = () => {
+        fetch("https://rancid-tomatillos.herokuapp.com/api/v2//movies/")
+        .then(res => {
+           if(!res.ok) {
+               throw new Error('Failed to fetch movies')
+           }
+           return res.json()})
+        .then(moviesData => this.setState({
```

Comment:
I see that you're initial state for the 'movies' property is movieData.movies - how is that coming through on page load before the fetch has resolved?